src/CopyrighFixer/CopyrighFixer/configparser.cpp

```diff
@@ -0,0 +1,47 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "configparser.h"
+#include <quasarapp.h>
+
+
+namespace CopyrighFixer {
+ConfigParser::ConfigParser() {
+
+}
+
+bool ConfigParser::parseOptions(Config &conf) const {
+
+    if (QuasarAppUtils::Params::isEndable("sourceDir")) {
+        conf.setSourceDir(QuasarAppUtils::Params::getArg("sourceDir"));
+    } else {
+        QuasarAppUtils::Params::log("Error: Not option sourceDir.",
+                                    QuasarAppUtils::VerboseLvl::Error);
```

Comment:
I think are better solution is set default source dir if user forget sets it.
The Default source dir it is PWD or current work directory: so it is '.'